src/babashka/http_server.clj

```diff
@@ -165,6 +165,41 @@
    {:headers (merge {"Content-Type" (ext-mime-type (fs/file-name path))} headers)
     :body (fs/file path)}))
 
+(defn- parse-range-header [range-header]
+  (map #(when % (Long/parseLong %))
+       (-> range-header
+           (str/replace #"^bytes=" "")
+           (str/split #"-"))))
+
+(defn- read-bytes [f [start end]]
```

Comment:
This can probably be more efficiently done with RandomAccessFile as follows:

https://stackoverflow.com/a/45675392/6264

Reply:
I also wonder if there's any reflection going on in this interop. Not that it matters for bb, but for the clj case it might. Let's put `(set! *warn-on-reflection* true)` at the top of this ns.

Reply:
Cool! Did both. :slightly_smiling_face: